cli/src/main/resources/application.yml

```diff
@@ -44,3 +44,9 @@ hono:
   client:
     hostnameVerificationRequired: false
     trustStorePath: target/config/hono-demo-certs-jar/trusted-certs.pem
+
+---
+
+spring:
+  profiles: amqp-adapter-cli
```

Comment:
you should add some defaults to be used when running the application as AMQP adapter client, e.g. the default port to connect to ...

Reply:
ok